README.md

```diff
@@ -8,10 +8,55 @@ All private keys used in the library are for testing only and not used in any pr
 
 ## Prerequisites
 
-1. bitcoind
-2. A fatjar of the federate node
-2. [OPTIONAL] HSM signer. (only needed if an hsmConfigs configuration is specified)
-3. `node` and `npm` latest LTS version
+### bitcoind
+- download
+  Current pipeline in jenkins is running bitcoind 0.17, 0.18.1 works for local execution.
+  - Go to https://bitcoin.org/en/release/v0.18.1 in this page there is the link to the download sources
+  https://bitcoincore.org/bin/bitcoin-core-0.18.1/
+  Select the *.tar.gz file according to your OS (for MacOS sillicon [bitcoin-0.18.1-osx64.tar.gz](https://bitcoincore.org/bin/bitcoin-core-0.18.1/bitcoin-0.18.1-osx64.tar.gz))
+  - Decompress and copy the path to the folder.
+  - run in terminal `sudo cp <path to bitcoin-core folder>/bin/bitcoin* /usr/local/bin`
+  - create a folder called bitcoindata in your workspace
+  - run to test the bitcoind installation standalone `bitcoind -deprecatedrpc=generate -addresstype=legacy -regtest -printtoconsole -server -rpcuser=rsk -rpcpassword=rsk -rpcport=18332 -txindex -datadir=<path to bitcoindata> $@`
```

Comment:
```bash
   curl -O https://bitcoin.org/bin/bitcoin-core-0.18.1/bitcoin-0.18.1-osx64.tar.gz
   tar -zxf bitcoin-0.18.1-osx64.tar.gz
   sudo mkdir -p /usr/local/bin
   sudo cp bitcoin-0.18.1/bin/bitcoin* /usr/local/bin/.
   rm -rf bitcoin-0.18.1*
   As a validation, you can run *bitcoind -daemon*. Run *bitcoin-cli stop* afterwards.
```

Reply:
Thank you, I'll adjust the instructions.